### ISSUE TITLE
Accuracy not Improving as in Paper Results 

### ISSUE THREAD

**Issue by meher-mankikar** [Horea94/Fruit-Images-Dataset#27]
Hello, 
I cloned the repo and am running the training file, but my accuracy seems to be improving much slower than the results presented in the paper. Are there some hyperparameter settings that should be changed to reproduce your results? I have attached some partial training results I am getting right now. 

Thank you for your help!
<img width="831" alt="Screenshot 2024-05-02 at 3 16 49 PM" src="https://github.com/Horea94/Fruit-Images-Dataset/assets/72577041/56e8efc4-bdef-4fb3-bb3a-cb5678592f63">

**Comment by meher-mankikar** [comment]
I was using the latest version for the results above. If using tf 1.8, were you using the code at image_classification_tf_1.8.0/network/fruit_train_net.py for training? 
I was originally running image_classification/Fruits-360 CNN.py but I believe this is only compatible with tf 2.0?

**Comment by Horea94** [comment]
Yes. However, that version is deprecated and I would still recommend using the one for tf 2.0 as it should achieve comparable performance even if the training progress differs.
